docs/features/software-catalog/system-model.md

```diff
@@ -108,8 +108,3 @@ domain would come with some documentation on how to accept payments for a new
 product or use-case, share the same entity types in their APIs, and integrate
 well with each other. Other domains could be “Content Ingestion”, “Ads” or
 “Search”.
-
-## Links
```

Comment:
The original RFC is linked higher in the page, and the second link seems overcome by the next page in the docs (YAML file format doc, rather than the original ADR).